modules/cli/test-scripts/streams-tests.sh

```diff
@@ -172,23 +225,109 @@ function streams_create {
     rc=$?
     # We expect a return code of 0 because this is a properly formed streams get command.
     if [[ "${rc}" != "0" ]]; then 
-        error "Failed to create stream with name and value used."
+        error "Failed to get stream with name $stream_name."
         exit 1
     fi
 
     # Check that the previous properties set created a stream, with the correct values
-    cat $output_file | grep "Galasa" -q -E
-    cat $output_file | grep "mystream" -q -E
-    cat $output_file | grep "enabled" -q -E
+    grep -q -E "Galasa" "$output_file"
+    rc1=$?
+
+    grep -q -E "$stream_name" "$output_file"
+    rc2=$?
+
+    grep -q -E "enabled" "$output_file"
+    rc3=$?
+
+    # Check if any of the greps failed
+    if [[ "$rc1" != "0" || "$rc2" != "0" || "$rc3" != "0" ]]; then
+        # Report which specific check failed
+        [[ "$rc1" != "0" ]] && error "Missing 'Galasa' in stream output"
+        [[ "$rc2" != "0" ]] && error "Missing '$stream_name' in stream output"
+        [[ "$rc3" != "0" ]] && error "Missing 'enabled' in stream output"
+        error "Stream successfully created, but values incorrect."
+        exit 1
+    fi
```

Comment:
We could check for errors directly after running each command instead of having to figure out which command failed?